app/views/shared/_links.haml

```diff
@@ -1,8 +1,8 @@
-%li= link_to 'diasporafoundation.org', "https://diasporafoundation.org"
-%li= link_to 'Wiki', "https://wiki.diasporafoundation.org"
-%li= link_to t('layouts.application.whats_new'), changelog_url
-%li= link_to t('layouts.header.code') + " " + pod_version, "#{source_url}", {:title => t('layouts.application.source_package')}
+%li= link_to 'diaspora*', "https://diasporafoundation.org", :target => "_blank"
+%li= link_to 'wiki', "https://wiki.diasporafoundation.org"
+%li= link_to 'uptime', "http://status.librenet.gr", :target => "_blank"
 %li= link_to t("layouts.application.statistics_link"), statistics_path
 %li= link_to(t('layouts.application.toggle'), toggle_mobile_path)
+%li= link_to 'source code', "https://github.com/librenet/librenet.gr", :target => "_blank"
```

Comment:
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Use the new Ruby 1.9 hash syntax.